common/lib/xmodule/xmodule/modulestore/xml.py

```diff
@@ -706,7 +706,7 @@ def make_course_key(self, org, course, run):
         """
         return CourseLocator(org, course, run, deprecated=True)
 
-    def get_courses(self, depth=0, **kwargs):
+    def get_courses(self, **kwargs):
```

Comment:
I found this hard to believe at first, but looking at the function signature for other definitions of `get_courses`, `(self, **kwargs)` seems to actually match what we already have elsewhere.